package.json

```diff
@@ -35,13 +35,16 @@
     "lint-react-test": "eslint --ext .js,.jsx -c .eslintrc-react-test src test/client",
     "lint": "npm run lint-node && npm run lint-react && npm run lint-react-test",
     "test-frontend": "node node_modules/karma/bin/karma start karma.conf.js",
-    "test-frontend-ci": "node node_modules/karma/bin/karma start --browsers PhantomJS,Firefox karma.conf.js",
+    "test-frontend-ci": "node node_modules/karma/bin/karma start --browsers PhantomJS,Firefox karma.conf.coverage.js",
+    "test-frontend-cov": "node node_modules/karma/bin/karma start karma.conf.coverage.js",
     "test-frontend-dev": "node node_modules/karma/bin/karma start karma.conf.dev.js",
     "test": "npm run test-frontend && echo 'TODO Server 13, Integration 12 tests.'",
     "test-ci": "npm run test-frontend-ci",
```

Comment:
cc/ @ryan-roemer do we need both `test-frontend-ci` and `test-ci`?  Is this future-proofing for when we have `test-server-ci` or something?